openshift/release/knative-eventing-channelbroker-ci.yaml

```diff
@@ -160,3 +160,165 @@ spec:
           containerPort: 9090
         - name: profiling
           containerPort: 8008
+---
```

Comment:
I dont think that `make RELEASE...` is needed.

if so, please delete the yamls before (there will be no diff)

Reply:
done